storage/innobase/log/log0log.cc

```diff
@@ -56,6 +56,7 @@ Created 12/9/1995 Heikki Tuuri
 #include "trx0roll.h"
 #include "srv0mon.h"
 #include "sync0sync.h"
+#include "row0log.h"
```

Comment:
Sorry if I asked this before, but why this? There are new declarations in row0log.h but none appear to be moved from somewhere else so that otherwise untouched source file starts needing them